Formula/chapel.rb

```diff
@@ -1,8 +1,8 @@
 class Chapel < Formula
   desc "Programming language for productive parallel computing at scale"
   homepage "https://chapel-lang.org/"
-  url "https://github.com/chapel-lang/chapel/releases/download/1.30.0/chapel-1.30.0.tar.gz"
-  sha256 "8d933f4b6e497d9699a14deedd222e18c77c523483957d66201731d0d94285d2"
+  url "https://chapel-lang.org/tmp/chapel-1.31.0.tar.gz"
```

Comment:
would this url be changed when doing the actual release?

Reply:
Yes, it will change.